op-interop-mon/monitor/metric_collector_test.go

```diff
@@ -220,44 +220,117 @@ func TestCollectMetrics(t *testing.T) {
 
 	tests := []testCase{
 		{
-			name:                         "empty job maps",
-			updater1Jobs:                 map[JobID]*Job{},
-			updater2Jobs:                 map[JobID]*Job{},
-			updater3Jobs:                 map[JobID]*Job{},
-			expectedMessageStatusCalls:   []expectedMessageStatusCall{},
-			expectedTerminalCalls:        []expectedTerminalCall{},
-			expectedExecutingRangeCalls:  []expectedBlockRangeCall{},
-			expectedInitiatingRangeCalls: []expectedBlockRangeCall{},
+			name:         "empty job maps",
+			updater1Jobs: map[JobID]*Job{},
+			updater2Jobs: map[JobID]*Job{},
+			updater3Jobs: map[JobID]*Job{},
+			expectedMessageStatusCalls: []expectedMessageStatusCall{
+				{executingChainID: "1", initiatingChainID: "1", status: "valid", count: 0},
```

Comment:
I take your point about having test data verbose and verifiable vs using helpers. I'm not 100% sure what is best here, but I do worry about this pattern making new tests quite hard to write. 

One idea I had was that we could write `{"1", "1", "valid", 0}` to at least reduce the verbosity of these tests.

Or, going further, we have the test loop over all combinations of the 3 labels, and assert a call with `count: 0` _unless_ the test case specifies a nonzero count (in which case we sub in the test case specified count). Then we can remove all the `count: 0` declarations and have those implicit.